core/src/sdk/README.md

```diff
@@ -0,0 +1,103 @@
+### Build C++ sdk
+
+The C++ sdk source code is under milvus/core/src/sdk. Build entire milvus project will also build the sdk project. 
```

Comment:
The C++ SDK source code is located under milvus/core/src/sdk. It is automatically built when you build the milvus project.